### ISSUE TITLE
Game Time

### ISSUE THREAD

**Issue by Bobcat90** [Ertego/gog-galaxy-itch.io#19]
When you click connect on the integration screen in Galaxy, the pop up says that game time is tracked as one of the features with this integration, but it doesn't seem to transfer time played from the itch client. I think it could be possible though, because when you click on a game in the itch client, there's a panel at the bottom showing hours played and last played. 

I've attached a screenshot showing this: ![image](https://user-images.githubusercontent.com/56528183/81691829-65d7e480-9455-11ea-9ccf-cc41dcbe6b43.png)

**Comment by lele-blue** [comment]
You are totally right, that's an oversight. The integration keeps track of the time on it's own. I'll try to implement this together with #6.
